packages/types/src/profiling.ts

```diff
@@ -1,5 +1,22 @@
 import type { DebugImage } from './debugMeta';
 import type { MeasurementUnit } from './measurement';
+import type { Integration } from './integration';
```

Comment:
These were moved to the types package so that once we port browser profiling to cont profiling we can adopt the same interface